raijin_scripts/execute_sync/run

```diff
@@ -92,7 +92,7 @@ do
   ##################################################################################################
   set -x  # echo is on
 
-  qsub -V -N dea-sync -q ${QUEUE} -W umask=33 -l wd,walltime=20:00:00,mem=25GB,ncpus=16 -m ae \
+  qsub -V -N dea-sync -q ${QUEUE} -W umask=33 -l wd,walltime=20:00:00,mem=310GB,ncpus=80 -m ae \
```

Comment:
This is really wrong sorry. And actually, the existing line is submitting very inefficient jobs.

`dea-sync` only uses a Process Pool for parallelism, which means it cannot use more cores than exist on a single compute node.

The current code is wrong because it is submitting jobs requesting 16 CPUs, but the `dea-sync` option `-j 4` tells `dea-sync` to only use a pool of 4 processes.

We need to change them to be the same number, and less than or equal 16.

We should also test running with 4 processes vs 16 processes, to see whether it is at all efficient to request more processes.

Reply:
`dea-sync` was actually using one worker for `sync` process. Hence, updated the `qsub` argument to use only one `cpu`.